### PR TITLE
Loosen oauth2client lower bound for Python

### PR DESCRIPTION
**Pull request by bjwatson**
FYI @jonparrott and @dhermes

### CODE REVIEW

File: config/dependencies.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 auth:
   python:
-    version: '3.0.0'
+    version: '1.4.11'
```
Comment (dhermes) [inline config/dependencies.yml]: You might need to start at 2.0. There was a large breakage there, but if your code doesn't depend on it that isn't such a big deal.

**Comment by bjwatson** [comment]
@dhermes Done.

**Comment by dhermes** [comment]
@bjwatson SG.

**Comment by geigerj** [comment]
What was the context behind this change? I see that `google-cloud-python` is now on version 3: https://github.com/GoogleCloudPlatform/google-cloud-python/commit/28a6004796495329bb8c02fa400aec39e2bc8db2#diff-6512dd83a1839f99da393055ac54c10b

**Comment by dhermes** [comment]
@geigerj We're only on 3.0.0 because `google-gax` is

**Comment by bjwatson** [comment]
@geigerj See @jonparrott's [comment](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2457#issuecomment-250066030) for more context.

**Comment by bjwatson** [comment]
@geigerj I just pushed a version bump that I used this morning. I'm going to assume it's okay and merge, rather than waste your time with extra reviews.

**Pull request merged**